frontend/benefit/handler/src/hooks/useApplicationsQuery.ts

```diff
@@ -1,12 +1,24 @@
 import { BackendEndpoint } from 'benefit/handler/backend-api/backend-api';
 import useBackendAPI from 'benefit/handler/hooks/useBackendAPI';
 import { ApplicationData } from 'benefit/handler/types/application';
+import { useTranslation } from 'next-i18next';
 import { useQuery, UseQueryResult } from 'react-query';
+import showErrorToast from 'shared/components/toast/show-error-toast';
 
 const useApplicationsQuery = (
   status: string[]
 ): UseQueryResult<ApplicationData[], Error> => {
   const { axios, handleResponse } = useBackendAPI();
+  const { t } = useTranslation();
+
+  const handleError = (): void => {
```

Comment:
later this could be in separate file unless this is the only react-query in handler.

Reply:
true